### PR TITLE
docs: update componentImports link

### PR DESCRIPTION
**Pull request by santoshyadavdev**

### CODE REVIEW

**Review by MatanBobi — COMMENTED** [review]
Hi Santosh! 
Nice seeing you here 😍 Thanks for this.
I left a minor comment.

File: docs/angular-testing-library/examples.mdx [hunk 1]
```diff
@@ -115,7 +115,7 @@ describe('Counter', () => {
 })
 ```
 
-To override the an import of a standalone component for your component test, you can use the [`ɵcomponentImports` property](api.mdx#ɵcomponentImports]).
+To override the an import of a standalone component for your component test, you can use the [`componentImports` property](api.mdx#componentImports).
```
Comment (timdeschryver) [inline docs/angular-testing-library/examples.mdx:118]: 👋 I did a drive-by and spotted a typo that wasn't caused by this commit.
Thanks Santosh!

```suggestion
To override an import of a standalone component for your component test, you can use the [`componentImports` property](api.mdx#componentImports).
```

Comment (santoshyadavdev) [reply docs/angular-testing-library/examples.mdx]: Done

**Review by MatanBobi — APPROVED** [review]
Thanks! :)

**Comment by MatanBobi** [comment]
@all-contributors please add @santoshyadavdev for docs :)
